packages/atacama/atacama.0.0.2/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Modern, pure OCaml socket pool for Riot"
+description:
+  "Atacama is a modern, pure OCaml socket pool for Riot inspired by Thousand Island. It aims to be easy to understand and reason about, while also being at least as stable and performant as the alternatives."
+maintainer: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+authors: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+license: "MIT"
+tags: ["topics" "socket" "socket pool" "riot"]
+homepage: "https://github.com/leostera/atacama"
+bug-reports: "https://github.com/leostera/atacama/issues"
+depends: [
+  "ocaml" {>= "5.1"}
+  "riot" {>= "0.0.1"}
```

Comment:
```suggestion
  "riot" {>= "0.0.2"}
```
The following comes from riot, right?
```
#=== ERROR while compiling atacama.0.0.2 ======================================#
# context              2.2.0~alpha3 | linux/x86_64 | ocaml-base-compiler.5.1.0 | pinned(https://github.com/leostera/atacama/releases/download/0.0.2/atacama-0.0.2.tbz)
# path                 ~/.opam/5.1/.opam-switch/build/atacama.0.0.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p atacama -j 255 @install
# exit-code            1
# env-file             ~/.opam/log/atacama-7-44d4f7.env
# output-file          ~/.opam/log/atacama-7-44d4f7.out
### output ###
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc.opt -w -40 -g -bin-annot -I atacama/.atacama.objs/byte -I /home/opam/.opam/5.1/lib/angstrom -I /home/opam/.opam/5.1/lib/bigstringaf -I /home/opam/.opam/5.1/lib/iomux -I /home/opam/.opam/5.1/lib/ocaml/unix -I /home/opam/.opam/5.1/lib/ptime -I /home/opam/.opam/5.1/lib/ptime/clock/os -I /home/opam/.opam/5.1/lib/riot -I /home/opam/.opam/5.1/lib/stringext -I /home/opam/.opam/5.1/lib/telemetry -I /home/opam/.opam/5.1/lib/uri -no-alias-deps -open Atacama__ -o atacama/.atacama.objs/byte/atacama__Telemetry_.cmo -c -impl atacama/telemetry_.ml)
# File "atacama/telemetry_.ml", line 6, characters 28-52:
# 6 |   | Listening of { socket : Net.Socket.listen_socket }
#                                 ^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound type constructor Net.Socket.listen_socket
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc.opt -w -40 -g -bin-annot -I atacama/.atacama.objs/byte -I /home/opam/.opam/5.1/lib/angstrom -I /home/opam/.opam/5.1/lib/bigstringaf -I /home/opam/.opam/5.1/lib/iomux -I /home/opam/.opam/5.1/lib/ocaml/unix -I /home/opam/.opam/5.1/lib/ptime -I /home/opam/.opam/5.1/lib/ptime/clock/os -I /home/opam/.opam/5.1/lib/riot -I /home/opam/.opam/5.1/lib/stringext -I /home/opam/.opam/5.1/lib/telemetry -I /home/opam/.opam/5.1/lib/uri -no-alias-deps -open Atacama__ -o atacama/.atacama.objs/byte/atacama__Transport.cmo -c -impl atacama/transport.ml)
# File "atacama/transport.ml", line 8, characters 5-29:
# 8 |     (Net.Socket.listen_socket, [> `System_limit ]) Net.Socket.result
#          ^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound type constructor Net.Socket.listen_socket
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc.opt -w -40 -g -bin-annot -I atacama/.atacama.objs/byte -I /home/opam/.opam/5.1/lib/angstrom -I /home/opam/.opam/5.1/lib/bigstringaf -I /home/opam/.opam/5.1/lib/iomux -I /home/opam/.opam/5.1/lib/ocaml/unix -I /home/opam/.opam/5.1/lib/ptime -I /home/opam/.opam/5.1/lib/ptime/clock/os -I /home/opam/.opam/5.1/lib/riot -I /home/opam/.opam/5.1/lib/stringext -I /home/opam/.opam/5.1/lib/telemetry -I /home/opam/.opam/5.1/lib/uri -no-alias-deps -open Atacama__ -o atacama/.atacama.objs/byte/atacama.cmi -c -intf atacama/atacama.mli)
# File "atacama/atacama.mli", line 97, characters 8-32:
# 97 |       ( Net.Socket.listen_socket,
#              ^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound type constructor Net.Socket.listen_socket
```

Reply:
Yup! I am bumping this to 0.0.3 now that that riot release is out.